### DISCUSSION TITLE
Export to PunyInform

### DISCUSSION THREAD

**Discussion by fredrikr** [JasonLautzenheiser/trizbort#572] [Ideas]
It would be nice to be able to export to PunyInform format.

The generated code for most rooms and objects would be identical as for regular Inform 6.

PunyInform has a lot of optional features, e.g. the library can be asked to consider light and dark, or not. If darkness is disabled, the light attribute isn't defined, and so can't be part of the room definitions. PunyInform also has a simpler way of defining doors, but some authors may not want to use it. Ideally, Trizbort would have a tab for PunyInform settings in Map Properties, to allow the author to get the kind of source they want. If that's not an option, let's settle for a set of options that will be the best default.

I'd be happy to cooperate to make this good.

**Comment by JasonLautzenheiser** [comment]
Sorry for the slow reply on this.   Been out of commission for awhile.   I like this idea.   I'll perhaps get something quick with some default settings in place, then work on getting some options setup for the user to adjust.
